### PR TITLE
All Ratification Polls for September 13

### PR DESCRIPTION
**Pull request by Hipogeo**
@Prose11

### CODE REVIEW

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:2]: Remove "Add"

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+---
+title: Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:2]: Remove "Add"

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Add Collateral Engineering Services Core Unit, CES-001 being accepted and implemented in the Maker Protocol.
+
+### Add Collateral Engineering Services Core Unit
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:28]: Maybe "Collateral Engineering Services Core Unit - Mandate"

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Add Development & UX Core Unit, DUX-001 being accepted and implemented in the Maker Protocol.
+
+### Add Development & UX Core Unit
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:28]: "Development & UX Core Unit - Mandate"

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/597512747641658bc38c72bc17d1c84e43153c2b/MIP39/MIP39c2-Subproposals/MIP39c2-SP20.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP20)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp20-adding-collateral-engineering-services-core-unit-ces-001/9809)
+
+### Add Core Unit Budget, CES-001
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Building upon the theme of sustainability of the Collateral Engineering Services Core Unit (CES), the budget reflects support of the Mandates and business continuity in MIP39. Any elements of this budget reflecting a likeness to other Core Units is a principle of building on the shoulders of giants. Much appreciation is given to all the trailblazers.
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:43]: Will have to add a * at the end of line. Every new line will need this for markdown, you can use the preview option to see if your italics are working

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+The CES budget is designed with the following in mind:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:44]: Add * to start and end to italicize

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+Paying for the operational costs to run the Core Unit
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:46]: Add * to start and end

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+Building a reserve for sustainability*
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:47]: Add * to start

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Our vision is to provide the best governance user experience following key governing principles such as accessibility, neutrality, and transparency. We want to do so by:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:32]: Italics, add * at the end

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+Facilitating an optimal decision-making process, allowing routine decisions to be executed in a fast and frictionless manner, while complex or controversial decisions are thoughtfully deliberated.
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:34]: Italics for section

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/597512747641658bc38c72bc17d1c84e43153c2b/MIP39/MIP39c2-Subproposals/MIP39c2-SP18.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP18)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp18-adding-development-ux-core-unit-dux-001/9770)
+
+### Add Core Unit Budget, DUX-001
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP#1 adds the budget for Core Unit DUX-001: Development and UX. It contains:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:54]: italics for section

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/597512747641658bc38c72bc17d1c84e43153c2b/MIP40/MIP40c3-Subproposals/MIP40c3-SP26.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP26)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp26-development-ux-core-unit-budget-dux-001/9774)
+
+### Add Core Unit MKR Budget, DUX-001
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP27 adds the MKR Incentive Plan budget for Core Unit DUX-001: Development and UX. It contains:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:69]: Italics

File: governance/polls/Ratification Poll for Add StarkNet Engineering Core Unit, SNE-001 - September 13, 2021.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
+---
+title: Ratification Poll for Add StarkNet Engineering Core Unit, SNE-001 - September 13, 2021
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add StarkNet Engineering Core Unit, SNE-001 - September 13, 2021.md:2]: Remove "Add"

File: governance/polls/Ratification Poll for Add StarkNet Engineering Core Unit, SNE-001 - September 13, 2021.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Add StarkNet Engineering Core Unit, SNE-001 being accepted and implemented in the Maker Protocol.
+
+### Add StarkNet Engineering Core Unit
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add StarkNet Engineering Core Unit, SNE-001 - September 13, 2021.md:28]: "StarkNet Engineering Core Unit - Mandate"

File: governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
+---
+title: Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md:2]: Remove "Add"

File: governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Add Strategic Happiness Core Unit, SH-001 being accepted and implemented in the Maker Protocol.
+
+### Add Strategic Happiness Core Unit
```
Comment (prose11) [inline governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md:28]: Strategic Happiness Core Unit - Mandate

File: governance/polls/Ratification Poll for GovAlpha Core Unit Budget Q4 2021 to Q1 2022 (MIP40c3-SP29) - September 13, 2021.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for GovAlpha Core Unit Budget Q4 2021 to Q1 2022 (MIP40c3-SP29) - September 13, 2021
+summary: MIP40c3-SPX adds the budget for Core Unit GOV-001: GovAlpha for October 2021 through March 2022.
```
Comment (prose11) [inline governance/polls/Ratification Poll for GovAlpha Core Unit Budget Q4 2021 to Q1 2022 (MIP40c3-SP29) - September 13, 2021.md:3]: Add MIP number

File: governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition toModify Core Unit Budget - Sustainable Ecosystem Scaling being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SPXX modifies the Dai budget for Core Unit SES-001: Sustainable Ecosystem Scaling. It contains:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md:30]: Fix italics

File: governance/polls/Ratification Poll for Risk Core Unit MKR Compensation (MIP40C3-SP25) -  September 13, 2021.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Risk Core Unit MKR Compensation being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This proposal contains:
```
Comment (prose11) [inline governance/polls/Ratification Poll for Risk Core Unit MKR Compensation (MIP40C3-SP25) -  September 13, 2021.md:30]: Fix italics

**Review by prose11 — CHANGES_REQUESTED** [review]
Nice Work, couple common fixes needed across most of these

File: governance/polls/Ratification Poll for Risk Core Unit MKR Compensation (MIP40C3-SP25) -  September 13, 2021.md (hunk 19 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Risk Core Unit MKR Compensation (MIP40C3-SP25) -  September 13, 2021.md:30]: Done!

File: governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md (hunk 18 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md:30]: Done!

File: governance/polls/Ratification Poll for GovAlpha Core Unit Budget Q4 2021 to Q1 2022 (MIP40c3-SP29) - September 13, 2021.md (hunk 17 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for GovAlpha Core Unit Budget Q4 2021 to Q1 2022 (MIP40c3-SP29) - September 13, 2021.md:3]: Done!

File: governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md (hunk 16 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md:28]: Done!

File: governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md (hunk 15 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Add Strategic Happiness Core Unit, SH-001 - September 13, 2021.md:2]: Done!

File: governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md (hunk 3 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md:28]: Done for all of them!

File: governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md (hunk 2 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Add Development & UX Core Unit, DUX-001 - September 13, 2021.md:2]: Done!

File: governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md:30]: Done!

File: governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
+---
+title: Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021
+summary: MIP40c3-SPXX modifies the Dai budget for Core Unit SES-001: Sustainable Ecosystem Scaling, replacing MIP40c3-SP10.
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Ratification Poll for Modify Core Unit Budget - Sustainable Ecosystem Scaling (MIP40c3-SP31) - September 13, 2021.md:3]: Done!

**Pull request merged**